apps/laboratory/src/components/Solana/SolanaTests.tsx

```diff
@@ -48,10 +49,16 @@ export function SolanaTests() {
           </Box>
           <Box>
             <Heading size="xs" textTransform="uppercase" pb="2">
-              Sign and Send Transaction
+              Sign and Send Transaction (dApp)
             </Heading>
             <SolanaSendTransactionTest />
           </Box>
+          <Box>
+            <Heading size="xs" textTransform="uppercase" pb="2">
+              Sign and Send Transaction (Wallet)
+            </Heading>
+            <SolanaSignAndSendTransaction />
+          </Box>
```

Comment:
What is the difference between Dapp and Wallet?

Reply:
Dapp: the dapp request to wallet sign transaction and sends to blockchain
Wallet: the dapp request to wallet sign and send the transaction

Reply:
Hmm I feel this is confusing, so you are doing basically

> Request wallet to sign transaction => Manually send it from lab
> Request wallet to sign and send transaction directly

Why send the tx from lab in the first place? isn't just requesting the signing enough? 

Just a nit tho

Reply:
There is the two ways of doing it and both are valid: the first case is testing the `signTransaction` and `sendTransaction` functions, the second is testing `signAndSendTransaction` which sends for wallet handle the blockchain connection.